presto-main/src/test/java/com/facebook/presto/sql/relational/TestLogicalRowExpressions.java

```diff
@@ -158,6 +152,92 @@ public void testPushNegationToLeaves()
         assertEquals(logicalRowExpressions.pushNegationToLeaves(not(compare(a, LESS_THAN_OR_EQUAL, not(not(b))))), compare(a, GREATER_THAN, b));
     }
 
+    @Test
+    public void testEliminateConstant()
+    {
+        // Testing eliminate constant
+        assertEquals(logicalRowExpressions.convertToConjunctiveNormalForm(or(and(TRUE_CONSTANT, a), and(FALSE_CONSTANT, b))),
```

Comment:
nit: I would put one new line for each parameter, i.e.

```java
        assertEquals(
               logicalRowExpressions.convertToConjunctiveNormalForm(or(and(TRUE_CONSTANT, a), and(FALSE_CONSTANT, b))),
                a);
```

ditto for other tests